lib/hammer_cli/i18n/find_task.rb

```diff
@@ -30,6 +32,20 @@ def define
           task :find => [:setup] do
             Rake::Task["gettext:po:update"].invoke
           end
+
+          desc 'Check languages with 50% or more coverage and create needed files'
+          task :find_new => [:setup] do
+            client = HammerCLI::TaskHelper::I18n::TxApiClient.new(domain: @domain)
+            stats = client.language_stats_collection
```

Comment:
If the user supplies wrong credentials, this somewhat unexpectedly returns an error hash
```
> client.language_stats_collection
=> {"errors"=>[{"status"=>"401", "code"=>"Unauthorized", "title"=>"Authentication failed", "detail"=>"Invalid token."}]}
```

leading to `undefined method `each_with_object' for nil:NilClass` on L41